chainercv/experimental/links/model/fcis/fcis_resnet101.py

```diff
@@ -0,0 +1,234 @@
+        self.group_size = group_size
+        self.roi_size = roi_size
+        self.loc_normalize_mean = loc_normalize_mean
+        self.loc_normalize_std = loc_normalize_std
+        self.iter2 = iter2
+
+        initialW = chainer.initializers.Normal(0.01)
+        with self.init_scope():
+            self.psroi_conv1 = L.Convolution2D(
+                2048, 1024, 1, 1, 0, initialW=initialW)
+            self.psroi_conv2 = L.Convolution2D(
+                1024, group_size * group_size * n_class * 2,
+                1, 1, 0, initialW=initialW)
+            self.psroi_conv3 = L.Convolution2D(
+                1024, group_size * group_size * 2 * 4,
+                1, 1, 0, initialW=initialW)
+
+    def __call__(self, x, rois, roi_indices, img_size):
+        h = F.relu(self.psroi_conv1(x))
+        h_cls_seg = self.psroi_conv2(h)
+        h_ag_locs = self.psroi_conv3(h)
+
+        # PSROI pooling and regression
+        roi_seg_scores, roi_ag_locs, roi_cls_scores = self._pool(
+            h_cls_seg, h_ag_locs, rois, roi_indices)
+        if self.iter2:
+            # 2nd Iteration
+            # get rois2 for more precise prediction
+            roi_ag_locs = roi_ag_locs.array
+            mean = self.xp.array(self.loc_normalize_mean)
+            std = self.xp.array(self.loc_normalize_std)
+            roi_loc = roi_ag_locs[:, 1, :]
+            roi_loc = (roi_loc * std + mean).astype(np.float32)
+            rois2 = loc2bbox(rois, roi_loc)
+
+            rois2[:, 0::2] = self.xp.clip(rois2[:, 0::2], 0, img_size[0])
+            rois2[:, 1::2] = self.xp.clip(rois2[:, 1::2], 0, img_size[1])
+
+            # PSROI pooling and regression
+            roi_seg_scores2, roi_ag_locs2, roi_cls_scores2 = self._pool(
+                h_cls_seg, h_ag_locs, rois2, roi_indices)
+
+            # concat 1st and 2nd iteration results
+            rois = self.xp.concatenate((rois, rois2))
+            roi_indices = self.xp.concatenate((roi_indices, roi_indices))
+            roi_seg_scores = F.concat(
+                (roi_seg_scores, roi_seg_scores2), axis=0)
+            roi_ag_locs = F.concat(
+                (roi_ag_locs, roi_ag_locs2), axis=0)
+            roi_cls_scores = F.concat(
+                (roi_cls_scores, roi_cls_scores2), axis=0)
+        return roi_seg_scores, roi_ag_locs, roi_cls_scores, rois, roi_indices
+
+    def _pool(
+            self, h_cls_seg, h_ag_locs, rois, roi_indices):
+        # PSROI Pooling
+        # shape: (n_rois, n_class*2, roi_size, roi_size)
+        pool_cls_seg = psroi_pooling_2d(
+            h_cls_seg, rois, roi_indices,
+            self.n_class * 2, self.roi_size, self.roi_size,
+            self.spatial_scale, self.group_size)
+        # shape: (n_rois, n_class, 2, roi_size, roi_size)
+        pool_cls_seg = pool_cls_seg.reshape(
+            (-1, self.n_class, 2, self.roi_size, self.roi_size))
+        # shape: (n_rois, 2*4, roi_size, roi_size)
+        pool_ag_locs = psroi_pooling_2d(
```

Comment:
May I suggest to name `(n_roi, 2 * 4, R, R)` as `roi_seg_ag_locs` instead of `pool_ag_locs`?
This seems like an consistent usage of `_seg_`, which is used for all variables with `(n_roi, *, R, R)`.